test/errorshow.jl

```diff
@@ -745,6 +747,14 @@ let err_str
     @test count(==("Maybe you forgot to use an operator such as *, ^, %, / etc. ?"), split(err_str, '\n')) == 1
 end
 
+let err_str
+    a = [1 2; 3 4]; 
+    err_str = @except_str (a[1][2] = 5) MethodError
+    @test occursin("\nAre you trying to index into an array? Separate the indices with commas: ", err_str)
```

Comment:
```
Error in testset errorshow:
2024-02-12 22:44:25 UTC	Test Failed at /Users/julia/.julia/scratchspaces/a66863c6-20e8-4ff4-8a62-49f30b1f605e/agent-cache/default-honeycrisp-R17H3W25T9.0/build/default-honeycrisp-R17H3W25T9-0/julialang/julia-master/julia-c8330b539a/share/julia/test/errorshow.jl:753
2024-02-12 22:44:25 UTC	  Expression: occursin("\nAre you trying to index into an array? Separate the indices with commas: ", err_str)
2024-02-12 22:44:25 UTC	   Evaluated: occursin("\nAre you trying to index into an array? Separate the indices with commas: ", "MethodError: no method matching setindex!(::Int64, ::Int64, ::Int64)\nThe function `setindex!` exists, but no method is defined for this combination of argument types.\nAre you trying to index into an array? For multi-dimensional arrays, separate the indices with commas: a[1, 2] rather than a[1][2]")
```

... For multi-dimensional arrays, ...